Trading/Exchange/okex_websocket_feed/okex_websocket.py

```diff
@@ -22,13 +22,10 @@ class OKExCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
     REQUIRED_ACTIVATED_TENTACLES = []
     EXCHANGE_FEEDS = {
         Feeds.TRADES: cryptofeed_constants.TRADES,
-        Feeds.KLINE: Feeds.UNSUPPORTED.value,
```

Comment:
we don't need to declare them as unsupported anymore ?

Reply:
Yes, with the new integration it's not necessary